hivemq-edge/src/frontend/src/__test-utils__/adapters/http.ts

```diff
@@ -0,0 +1,152 @@
+import { ProtocolAdapter } from '@/api/__generated__'
```

Comment:
Are these files auto-generated?

Reply:
Yes, using a manual converter. They are not needed anymore. Removing them